domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/Søknadsbehandling.kt

```diff
@@ -1773,6 +2062,7 @@ sealed class KunneIkkeIverksette {
     object FantIkkePerson : KunneIkkeIverksette()
     object FikkIkkeHentetSaksbehandlerEllerAttestant : KunneIkkeIverksette()
     object KunneIkkeGenerereVedtaksbrev : KunneIkkeIverksette()
+    object AvkortingErUfullstendig : KunneIkkeIverksette()
```

Comment:
2000 linjer and counting ja :) På tide og la denne bli splittet opp snart :grimacing: